src/relayclient/GSNConfigurator.ts

```diff
@@ -36,6 +36,7 @@ export const defaultGsnConfig: GSNConfig = {
   sliceSize: 3,
   relayTimeoutGrace: DEFAULT_RELAY_TIMEOUT_GRACE_SEC,
   methodSuffix: '',
+  requiredVersionRange: '^2.0.0',
```

Comment:
Why would we need to make it overridable though?
Among all configurable fields, at least defining the required version is kind of our responsibility IMO.

Reply:
possible future bugs, or troubleshooting.
I'm don't want to document it, since its basically internal (much like the relayLookupWindowBlocks)
there still a possibility that a client that wasn't supposed to support a contract, in some edge cases, is still usable.